<launch>
  <arg name="device_name"      default="/dev/ttyUSB0"/>
  <arg name="baud_rate"        default="1000000"/>
  <arg name="protocol_version" default="2.0"/>

  <arg name="pan_id"           default="1"/>

  <param name="device_name"      value="$(arg device_name)"/>
  <param name="baud_rate"        value="$(arg baud_rate)"/>
  <param name="protocol_version" value="$(arg protocol_version)"/>

  <param name="pan_id"           value="$(arg pan_id)"/>

  <node name="poli_pan_position_control" pkg="poli_pan_tilt" type="poli_pan_position_control" required="true" output="screen">
    <rosparam>
      profile_velocity: 100
      profile_acceleration: 10
    </rosparam>
  </node>
</launch>
